### ISSUE TITLE
Is this project abandoned?

### ISSUE THREAD

**Issue by GreatMichaelLee** [ntop/n2n#1058]
Seems lost maintenance....more than 4 months no new feature or fix code merged. Is there any fix or roadmap feature WIP still?

**Comment by hamishcoleman** [comment]
No, it is not abandoned.  We still have a number of roadmap tickets with plans for the future, but the developers have been busy recently and not been able to work on them.  We are still responding to issues and committing fixes where they are the result of bugs.

Is there something in particular you are waiting on?

**Comment by GreatMichaelLee** [comment]
> No, it is not abandoned. We still have a number of roadmap tickets with plans for the future, but the developers have been busy recently and not been able to work on them. We are still responding to issues and committing fixes where they are the result of bugs.
> 
> Is there something in particular you are waiting on?

thanks for the update, nice to hear that. Emmm.... I am acturally waiting for something more easier way to maintenance the federation topology between supernodes. Say automatically sync up the edge info under a federation between supernodes from a source(master) supernode so that they will no need to do the edge information maintenance job by manual in each supernode. if control signal panel cannot reach so far, at least a CLI option looks feel better comparing to the configure file editing method which used for now.

Also, other 3.2/4.0 features are highly being waited as well:)

Anyway, thanks.

**Comment by hamishcoleman** [comment]
Thats not an improvement we have been tracking - and I'm not sure if I completely understand.  Are you able to explain further?  I'm not sure if you are trying to propagate supernode configuration changes, or have edges get this information.

**Comment by GreatMichaelLee** [comment]
> 

Acutrally I mean the edge autodiscovery and configure sync up between supernodes, say I would only like to configure the edge name/phase-key etc on the master(active) supernode, and once the master(active) supernode establish the communication with the other slave(standby) supernode, it could automatically sync up those edge configuration information to those standby supenrode and no need any manuel job to do it again whatever it copy to there or other way. In one word, configure edge in one point,one time, all supernodes would know the edge topology configure and synced up automatically.

**Comment by Logan007** [comment]
What exact part of configuration do you copy to standby-supernode?

**Comment by GreatMichaelLee** [comment]
> What exact part of configuration do you copy to standby-supernode?

community file

**Comment by sweihub** [comment]
Let sing high praise for N2N, awsome project, the peer-to-peer network is a high speed network.

But the network is sometimes unstable, I have to schedule a daily restart for both supernode & edge.
